portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,8 +37,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     ArrayList<String> comments = new ArrayList<String>();  
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    int limit = getCommentLimit(request);
     PreparedQuery results = datastore.prepare(query);
-    for (Entity entity : results.asIterable()) {
+    List<Entity> commentList = results.asList(FetchOptions.Builder.withLimit(limit)); 
```

Comment:
Just curious -- if you use the default -1 here, does it show all the comments, or none?
If its all, it might be safer to use a different default

Reply:
If I use -1 it doesn't show any comments!

Reply:
if you want it to display a default number of comments, declare a variable DEFAULT and pass it through when user inputs incorrect/no input